client/my-sites/site-settings/seo-settings/help.jsx

```diff
@@ -23,23 +26,26 @@ const SeoSettingsHelpCard = ( {
 
 	return (
 		<div>
-			<SectionHeader label={ translate( 'Search Engine Optimization' ) } />
-			<Card>
-				<p>
-					{ translate(
-						'{{b}}WordPress.com has great SEO{{/b}} out of the box. All of our themes are optimized ' +
-						'for search engines, so you don\'t have to do anything extra. However, you can tweak ' +
-						'these settings if you\'d like more advanced control. Read more about what you can do ' +
-						'to {{a}}optimize your site\'s SEO{{/a}}.',
-						{
-							components: {
-								a: <a href={ seoHelpLink } />,
-								b: <strong />
+			<SectionHeader label={ translate( 'Search engine optimization' ) } />
```

Comment:
It's a good suggestion, however we discussed going with sentence case for these, so I'll skip this one.